lib/players/human.rb

```diff
@@ -0,0 +1,11 @@
+module Players
+
+  class Human < Player
+
+    def move(board)
+      input=gets.chomp
+      input if board.valid_move?(input)
+    end
```

Comment:
The if statement here seems redundant.  You're already checking for valid input in your `Game#turn` method, so here all we need to do is ask the user to input, and return the response.  This method can get called again if the input is invalid:
``` ruby 
def move(board)
  puts "Make your move!"
  gets.chomp
end
```